### DISCUSSION TITLE
Code wont run at all on visual studio code.

### DISCUSSION THREAD

**Discussion by david2107** [community/community#21396] [Other Feature Feedback, Questions, & Ideas]
<p>Command “Run code” will not appear. Ctrl+Alt+N does nothing.</p>
<p>I went on the raw default settings and something was detected in the “Problems” window at the bottom:</p>
<p>[jsonc] End of file expected. [0] (346,1)</p>
<p>Can anyone help please?</p>

**Comment by david2107** [comment]
<p>Does anyone know how to remove the issue?</p>
<p>Ive figured it out myself.</p>

**Comment by david2107** [comment, marked as answer]
<p>Disabled extension was the issue I believe.</p>
<p>Solution(?): Ensure you have a suitable extension downloaded or enabled (?)</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/david2107">@david2107</a>, Thanks for coming back to share what worked for you! We’re looking forward to seeing you around!</p>
